### DISCUSSION TITLE
how to use keyboard shortcut inside？

### DISCUSSION THREAD

**Discussion by crysishacker** [sandboxie-plus/Sandboxie#4101] [Q&A Feedback]
For example, my software shortcut keys is ctrl alt d
it just not functional

**Comment by love-code-yeyixiao** [comment]
Try it.
`ProcessGroup=<HotKeyAllow>,ProgramName.exe`
`OpenWinClass=<HotKeyAllow>,*`

Although the tray will maybe hide.

**Reply by crysishacker** [reply]
is it just simply copy to “edit in section”？

**Reply by yeqwe** [reply]
me too，i have this problem

**Reply by love-code-yeyixiao** [reply]
> is it just simply copy to “edit in section”？

Modify "ProgramName.exe" to your nesscessary application.
